src/components/MangaControl.tsx

```diff
@@ -29,6 +29,7 @@ const MangaControl: React.FunctionComponent<Props> = (props: Props) => {
     updateReaderSettings({...readerSettings, colourTheme: event.currentTarget.value as ColourTheme });
   };
   const changeChapter = (event: React.ChangeEvent<HTMLSelectElement>) => {
+    console.log(event.currentTarget.value);
     updateChapter(+event.currentTarget.value);

```

Comment:
just make sure event.currentTarget.value is actually a number

Reply:
ok

Reply:
gdfg

Reply:
The thing is, changeChapter is called only when a chapter is selected, and the value of all those inputs are based on index, which is a number, so the error will never happen